interpolator/interpolator.F90

```diff
@@ -479,6 +509,62 @@ subroutine interpolator_init( clim_type, file_name, lonb_mod, latb_mod, &
 !  clim_type  :: An interpolate type containing the necessary file and field data to be passed to the interpolator routine.
 !  clim_units :: A list of the units for the components listed in data_names.
 !
+integer :: io, ierr
+logical :: the_file_exists
+
+if (.not. module_is_initialized) then
+  call fms_init
+  call diag_manager_init
+  call horiz_interp_init
+
+!--------------------------------------------------------------------
+! namelist input
+!--------------------------------------------------------------------
+
+the_file_exists = fms2_io_file_exist('input.nml')
+
+if (the_file_exists) then
+    read (input_nml_file, nml=interpolator_nml, iostat=io)
+    ierr = check_nml_error(io,'interpolator_nml')
+end if
```

Comment:
Should there also be internal namelist file read logic here?

Reply:
I thought we were working on getting rid of the `#ifdef INTERNAL_FILE_NML` because it was **_always_** set to true?

Reply:
You're right.  I misread the code.